package.json

```diff
@@ -1,37 +1,38 @@
 {
   "name": "openfin-service-tooling",
-  "version": "1.0.0",
+  "version": "1.0.2",
```

Comment:
Are you incrementing this on each commit? It should only be incremented on each release, and the module hasn't been published yet.

Reply:
I was supposed to set this back to 0 before push, whoops